workflows/repeatmasking/RepeatMasking-Workflow.ga

```diff
@@ -0,0 +1,168 @@
+            "label": null,
+            "name": "RepeatMasker",
+            "outputs": [
+                {
+                    "name": "output_masked_genome",
+                    "type": "fasta"
+                },
+                {
+                    "name": "output_log",
+                    "type": "tabular"
+                },
+                {
+                    "name": "output_table",
+                    "type": "txt"
+                },
+                {
+                    "name": "output_repeat_catalog",
+                    "type": "txt"
+                },
+                {
+                    "name": "output_gff",
+                    "type": "gff"
+                }
+            ],
+            "position": {
+                "left": 450,
+                "top": 10
+            },
+            "post_job_actions": {},
+            "tool_id": "toolshed.g2.bx.psu.edu/repos/bgruening/repeat_masker/repeatmasker_wrapper/4.1.5+galaxy0",
+            "tool_shed_repository": {
+                "changeset_revision": "ba6d2c32f797",
+                "name": "repeat_masker",
+                "owner": "bgruening",
+                "tool_shed": "toolshed.g2.bx.psu.edu"
+            },
+            "tool_state": "{\"__input_ext\": \"input\", \"advanced\": {\"is_only\": false, \"is_clip\": false, \"no_is\": false, \"rodspec\": false, \"primspec\": false, \"nolow\": false, \"noint\": false, \"norna\": false, \"alu\": false, \"div\": false, \"search_speed\": \"\", \"frag\": \"40000\", \"gc\": null, \"gccalc\": false, \"nocut\": false, \"xout\": false, \"keep_alignments\": false, \"invert_alignments\": false, \"poly\": false}, \"chromInfo\": \"/shared/ifbstor1/galaxy/mutable-config/tool-data/shared/ucsc/chrom/?.len\", \"excln\": true, \"gff\": true, \"input_fasta\": null, \"repeat_source\": {\"source_type\": \"dfam\", \"__current_case__\": 0, \"species_source\": {\"species_from_list\": \"no\", \"__current_case__\": 1, \"species_name\": \"\"}}, \"xsmall\": true, \"__page__\": null, \"__rerun_remap_job_id__\": null}",
+            "tool_version": "4.1.5+galaxy0",
+            "type": "tool",
+            "uuid": "e6c8e6a1-efe8-4291-b12b-5fdb3795b6ca",
+            "when": null,
+            "workflow_outputs": [
+                {
+                    "output_name": "output_masked_genome",
+                    "label": "output_masked_genome"
+                },
+                {
+                    "output_name": "output_log",
+                    "label": "output_log"
+                },
+                {
+                    "output_name": "output_table",
+                    "label": "output_table"
+                },
+                {
+                    "output_name": "output_repeat_catalog",
+                    "label": "output_repeat_catalog"
+                },
+                {
+                    "output_name": "output_gff",
+                    "label": "output_gff"
+                }
+            ]
```

Comment:
I think it would be nice to provide nicer labels for the outputs here ... since they are workflow outputs I don't think you need to prefix them with `output_`.